### ISSUE TITLE
PointsRendererWithFrags renders a smaller result than the ground truth

### ISSUE THREAD

**Issue by surheaven** [facebookresearch/pytorch3d#1535]
## ❓ Questions on how to use PyTorch3D

pytorch3d renders the mask smaller???.
py3.8_cuda11.3_cudnn8.2.0_0
pytorch3d: 0.4.0

code like this:

H,W=1080,1920
focals,princeple_ps,Rs,Ts,H,W=self.dataset.get_camera_parameters(frame_ids.numel(),device)
cameras=RectifiedPerspectiveCameras(focals,princeple_ps,Rs,Ts,image_size=[(W, H)]).to(device)
raster_settings_silhouette = PointsRasterizationSettings(
                                image_size=(H,W), 
				radius=0.0041,
				bin_size=(92 if max(H,W)>1024 and max(H,W)<=2048 else None),
				points_per_pixel=50,
				)   
self.pcRender=PointsRendererWithFrags(
			    rasterizer=PointsRasterizer(
                                cameras=cameras, 
				raster_settings=raster_settings_silhouette
				),
			    compositor=AlphaCompositor(background_color=None)
			).to(device)
features = [torch.ones(self.TmpVs.shape[0], 1, device=self.TmpVs.device) for _ in range(N)]
masks, frags = self.pcRender(Pointclouds(points=defMeshes.verts_list(), features=features))

I used the calibrated camera to render the mesh, but the mask I got was a turn smaller than gt_mask. Why is there such a problem?
The following figure shows the difference between mask and gt_mask.
![image](https://github.com/facebookresearch/pytorch3d/assets/89247297/81599fc2-5384-4828-b665-3e912b4532e5)
